src/EditorFeatures/Core/Implementation/InlineRename/InlineRenameSession.cs

```diff
@@ -144,7 +145,7 @@ private void InitializeOpenBuffers(SnapshotSpan triggerSpan)
 
                 foreach (var buffer in openBuffers)
                 {
-                    CreateOpenTextBufferManagerForBuffer(buffer, buffer.AsTextContainer().GetRelatedDocuments());
+                    TryPopulateOpenTextBufferManagerForBuffer(buffer, buffer.AsTextContainer().GetRelatedDocuments());
```

Comment:
We don't really care if this fails, although we could care on new line 164 below, except we'll never fail to index into the `_openTextBuffers` dictionary because we're using the triggering text buffer as the key which will always support rename because we explicitly block invoking inline rename from Venus buffers.

I could add an assert that the dictionary contains the key before we index, but I argue that if we encounter that scenario, something else is horribly broken and we'd want to know about it instead of swallowing the error.